homeassistant/components/climate/icons.json

```diff
@@ -38,7 +38,8 @@
             "comfort": "mdi:sofa",
             "eco": "mdi:leaf",
             "home": "mdi:home",
-            "sleep": "mdi:bed"
+            "sleep": "mdi:bed",
+            "none": "mdi:set-none"
```

Comment:
`none` is not one of our (historical) default presets. If an integration provides a `none` preset and wants to provide an icon for it, it can do so itself in the specific integration.

../Frenck

Reply:
`ClimateEntity` has `None` as a default preset so I think this PR was actually correct to implement an icon for it.
Ref: https://developers.home-assistant.io/docs/core/entity/climate#presets